docs/install_from_github_release.md

```diff
@@ -4,13 +4,43 @@ This document explains how to install Element X Android from a Github Release.
 
 <!--- TOC -->
 
-* [Requirements](#requirements)
-* [Steps](#steps)
-* [I already have the application on my phone](#i-already-have-the-application-on-my-phone)
+* [Installing the universal APK](#installing-the-universal-apk)
+  * [Instructions](#instructions)
+  * [Steps](#steps)
+  * [I already have the application on my phone](#i-already-have-the-application-on-my-phone)
+* [Installing from the App Bundle](#installing-from-the-app-bundle)
+  * [Requirements](#requirements)
+  * [Steps](#steps)
+  * [I already have the application on my phone](#i-already-have-the-application-on-my-phone)
 
 <!--- END -->
 
-## Requirements
+## Installing the universal APK
+
+### Instructions
+
+The easiest way to install the application from the application from a GitHub release is to use the universal APK which is attached to the release. This APK is compatible with all Android devices, but it is not optimized for any of them. So it may not be as fast as it could be on your device, and it may not be as small as it could be.
```

Comment:
```suggestion
The easiest way to install the application from a GitHub release is to use the universal APK which is attached to the release. This APK is compatible with all Android devices, but it is not optimized for any of them. So it may not be as fast as it could be on your device, and it may not be as small as it could be.
```